### PR TITLE
ImportManager: add a importFromS3 API call

### PR DESCRIPTION
**Pull request by LeoIannacone**
Ref issue: https://codio.myjetbrains.com/youtrack/issue/codio-6434

### CODE REVIEW

**Comment by destitutus** [comment]
tests?

**Comment by LeoIannacone** [comment]
added

File: test/import-manager.spec.js [hunk 1]
```diff
@@ -59,6 +59,33 @@ describe('ImportManager', function () {
                 });
             });
         });
+        describe('importFromS3', function () {
+            it('calls the correct request', function () {
+
+                return importManager.importFromS3('/path/to/file', {
```
Comment (AndreyNikitin) [inline test/import-manager.spec.js]: path start from / ?

Comment (LeoIannacone) [inline test/import-manager.spec.js]: Yes, '/' is not correct. But it will be cleaned server side

**Comment by AndreyNikitin** [comment]
:+1:

**Pull request merged**